Makefile

```diff
@@ -22,7 +22,7 @@ setup: dune-workspace
 
 .PHONY: exe
 exe: setup
-	dune build _build/dev/src/ocamlformat.exe _build/release/src/ocamlformat.exe
+	dune build _build/dev/src/ocamlformat.exe _build/release/src/ocamlformat.exe _build/release/src/ocamlformat-help.txt
```

Comment:
I'm not sure this should be added to the `exe` target. What about adding a `man` or `doc` or something target, and adding that target to `default`?